src/main/kotlin/com/openlattice/subscriptions/SubscriptionApi.kt

```diff
@@ -37,42 +36,29 @@ interface SubscriptionApi {
         const val ALL = "/all"
         const val ENTITY_KEY_ID = "entityKeyId"
         const val ENTITY_KEY_ID_PATH = "/{$ENTITY_KEY_ID}"
-
-        const val CONTACT = "contact"
-        const val CONTACT_INFO_PATH = "/$CONTACT"
-        const val ORGANIZATION = "organization"
-        const val ORGANIZATION_PATH = "/$ORGANIZATION"
-        const val ORGANIZATIION_ID_PATH = "/{$ORGANIZATION}"
     }
 
     /**
-     * Creates a subscription
-     */
-    @POST(BASE + ORGANIZATION_PATH + ORGANIZATIION_ID_PATH)
-    fun createOrUpdateSubscription(@Path(ORGANIZATION) organizationId: UUID, @Body subscription: NeighborhoodQuery)
-
-    /**
-     * Removes a subscription
+     * Removes a query
      */
     @DELETE(BASE + ENTITY_KEY_ID_PATH)
     fun deleteSubscription(@Path(ENTITY_KEY_ID) subscriptionId: UUID)
 
     /**
      * Create or Update Subscription contact info
      */
-    @POST(BASE + CONTACT_INFO_PATH)
-    fun createOrUpdateSubscriptionContactInfo(@Body contactInfo: SubscriptionContact)
+    @PUT(BASE )
```

Comment:
Can this be `@POST` instead (since it's an upsert)? 
/ update description

Reply:
I changed to put because it is idempotent and GET requests with bodies cause problems for a lot of tools. If we make this POST we'll have to merge the two GETs in one call. Totally happy to do this if people are okay with body containing GET (@UnsungHero97 @katherinebernstein )